src/Documentation/ListDocumentGenerator.php

```diff
@@ -93,7 +93,7 @@ static function (FixerInterface $fixer1, FixerInterface $fixer2): int {
 
                 foreach ($configurationDefinition->getOptions() as $option) {
                     $documentation .= "\n   - | ``{$option->getName()}``";
-                    $documentation .= "\n     | {$option->getDescription()}";
+                    $documentation .= "\n     | ".RstUtils::toRst($option->getDescription());
```

Comment:
This change fixes one thing, but unfortunately is breaking another... Currently we have:

<img width="924" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/66159d55-efd6-4673-ba8c-be975678f841">

After this change:

<img width="841" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/0928076f-8a77-4fb9-9ebf-5476b29b7d43">

As you can see it breaks formatting because of line wraps (new lines are not indented properly and not starting with `|`. We need to ensure proper formatting of whole file.